GitCliUtil.go

```diff
@@ -46,7 +46,13 @@ func (impl *GitUtil) runCommandWithCred(cmd *exec.Cmd, userName, password string
 }
 
 func (impl *GitUtil) runCommand(cmd *exec.Cmd) (response, errMsg string, err error) {
-	cmd.Env = append(cmd.Env, "HOME=/dev/null")
+	return impl.runCommandForSuppliedNullifiedEnv(cmd, true)
+}
+
+func (impl *GitUtil) runCommandForSuppliedNullifiedEnv(cmd *exec.Cmd, nullifiedHomeEnv bool) (response, errMsg string, err error) {
```

Comment:
based on function name it seems it runs commnd for NullifiedEnv only but we are taking nullifiedHomeEnv as argument, should change function name

Reply:
also argument name should also be isNullifiedHomeEnv

Reply:
Argument name is changed to - isNullifiedHomeEnv.
regarding function name - function name is runCommandFor**Supplied**NullifiedEnv.

Reply:
then variable name should be setHomeEnvToNull ?

Reply:
makes more sense. 
done